### ISSUE TITLE
Value of `b_end_partner_config.aws_config.name` is not respected

### ISSUE THREAD

**Issue by Kostoprav19** [megaport/terraform-provider-megaport#96]
Hello!

AWS Hosted Connection name is always "MEGAPORT" regardless of what I put in `b_end_partner_config.aws_config.name`.

```
resource "megaport_vxc" "aws" {
 product_name         = "TEST-AWS-VXC"
 rate_limit           = 100
 contract_term_months = 1


 a_end = {
   requested_product_uid = megaport_mve.fortigate.product_uid
   ordered_vlan          = 100
   vnic_index            = 1
 }


 b_end = {
   requested_product_uid = data.megaport_partner.aws_port.product_uid
 }


 b_end_partner_config = {
   partner = "aws"
   aws_config = {
     name          = "TEST-AWS"    # <---------------------
     asn           = 65121
     type          = "private"
     connect_type  = "AWSHC"
     amazon_asn    = 64512
     owner_account = "893677514652"
   }
 }
}
```

**Comment by mega-alex** [comment]
This should be fixed in v1.0.1 if you wouldn't mind giving that a try?

**Comment by Kostoprav19** [comment]
Thank you! Tested - now name is set properly. This issue can be closed.
